jest.config.ts

```diff
@@ -1,6 +1,10 @@
-import type { Config } from "jest";
+import type { JestConfigWithTsJest } from "ts-jest";
+import { pathsToModuleNameMapper } from "ts-jest";
 
-const config: Config = {
+import { compilerOptions } from "./tsconfig.json";
+
+const config: JestConfigWithTsJest = {
+  preset: "ts-jest",
```

Comment:
**question (non-blocking):** not sure if we really need this preset, I integrated also ts-paths in [this](https://github.com/AlbertHernandez/express-typescript-service-template/blob/22462ffce8ca4e68229e79862b870d4537db517a/jest.config.ts#L8) other template and this was not needed